lib/Define.ts

```diff
@@ -0,0 +1,65 @@
+import {
+    Message,
+    MessageEmbed,
+} from 'discord.js';
+
+import fetch from 'node-fetch';
+
+import {
+    Paginate,
+    DisplayType,
+} from './Paginate.js';
+
+
+export async function handleDefine(msg: Message, args: string): Promise<void> {
+    let query = args.trim();
+
+    /* Prevent leaking our IP */
+    if (query.includes('auto:ip')) {
+        query = '';
+    }
+
+    if (query === '') {
+        msg.reply(`No word given.`);
+        return;
+    }
+
+    query = query.replaceAll(" ", "%20");
+    
+    const url = `https://api.dictionaryapi.dev/api/v2/entries/en/${query}`;
+
+    try {
+
+        const res = await fetch(url);
+        const data = await res.json();
+
+        const embed = new MessageEmbed();
+        const meanings = data[0].meanings
+
+        
```

Comment:
```suggestion
```

avoid double blank newlines unless there's a particular reason to separate the sections of code